boltdesignsystem/bolt
# BDS-1573 action element schema

## Jira

http://vjira2:8080/browse/BDS-1573

## Summary

Add schema for BoltActionElement

## Details

- Create a schema to document BoltActionElement
- Uses the BoltActionElement schema in bolt-button (including removing usage of deprecated props from pl documentation)
- Update bolt-link schema to be js-based
- Uses the BoltActionElement schema in bolt-link (including removing usage of deprecated props from pl documentation)

## How to test

- Confirm that the schema docs for bolt-button and bolt-link both show the inherited props from BoltActionElement (including `href` and `url`)
- Confirm that the schema docs for bolt-link does not have a `disabled` prop (follow up ticket in http://vjira2:8080/browse/BDS-2072)
- Confirm no regressions (tests passing, etc-- this PR shouldn't modify functionality).
